pkg/component/controller/apiendpointreconciler.go

```diff
@@ -23,14 +23,16 @@ import (
 	"sort"
 	"time"
 
-	"github.com/sirupsen/logrus"
+	"github.com/k0sproject/k0s/pkg/apis/k0s.k0sproject.io/v1beta1"
```

Comment:
I'm all into this. Do you have some tool suggestion?

Reply:
I'm only familiar with [gci](https://github.com/daixiang0/gci), I think it's adecuate because it's included in one of the tests of [golangci-lint](https://github.com/golangci/golangci-lint/blob/15cba447fd0e577f1f60170bb188b5c0eca04dab/pkg/golinters/gci.go) and I believe we can add a test for this in make lint without adding further dependencies.

Reply:
I'd suggest we enforce some ordering in the linter settings and do not move force reformatting to the CI, because it would require setting up signed off for commits, introducing bot account with git commit & git push privileges, which seems like unnecessary hustle in my opinion.